apps/files/src/mixins.js

```diff
@@ -421,8 +421,16 @@ export default {
             })
           })
         } else {
+          const extraHeaders = {}
+          if (file.lastModifiedDate) {
+            extraHeaders['X-OC-Mtime'] = '' + file.lastModifiedDate.getTime() / 1000
+          }
+          if (file.lastModified) {
```

Comment:
Tiny comment: Since you're writing to the same header prop, you could do an `else if` here.